arangod/GeneralServer/HttpCommTask.cpp

```diff
@@ -119,6 +122,10 @@ int HttpCommTask<T>::on_url(llhttp_t* p, const char* at, size_t len) {
 
   me->_url.append(at, len);
   return HPE_OK;
+} catch (...) {
+  // the caller of this function is a C function, which doesn't have no
+  // idea about exceptions. we must not let an exception escape from here.
```

Comment:
```suggestion
  // the caller of this function is a C function, which doesn't know
  // exceptions. we must not let an exception escape from here.
```

Reply:
Code comments (this one and the following) adjusted as proposed.